dev/com.ibm.ws.microprofile.faulttolerance.cdi.2.1.services/bnd.bnd

```diff
@@ -0,0 +1,35 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-Name: com.ibm.ws.microprofile.faulttolerance.cdi.2.1.services
+Bundle-SymbolicName: com.ibm.ws.microprofile.faulttolerance.cdi.2.1.services
+Bundle-Description: MicroProfile Fault Tolerance CDI Integration, version ${bVersion} services
```

Comment:
Should the bundle description give the FT version?

Reply:
done